### ISSUE TITLE
Product Strategy Guide QR Code Link is broken

### ISSUE THREAD

**Issue by antonyjkx** [openpracticelibrary/carddecks#2]
In the open practice card, "Product Strategy Guide" link is not found. I don't find that in here https://openpracticelibrary.com/ also.

**Comment by bridmacks** [comment]
@tdbeattie this is the card which is referred in the issue, I cannot see a corresponding entry in the OPL, has this practice been removed? 

![Screenshot 2020-01-28 at 09 44 37](https://user-images.githubusercontent.com/49513727/73252568-d9a9f600-41b2-11ea-884a-94e38be068b9.png)

**Comment by tdbeattie** [comment]
@bridmacks yes, just looked into this and it looks like there was a PR https://github.com/openpracticelibrary/openpracticelibrary/pull/702 which merged this content into the Lean Canvas.

Therefore, we should remove the Product Strategy Guide from the card deck.

Separately, I will raise a GitHub Issue in the Open Practice Library repo to have a solution for how we gracefully handle old OPL pages (where people may have old versions of the card decks and still scan QR codes)

**Comment by tdbeattie** [comment]
Raised https://github.com/openpracticelibrary/openpracticelibrary/issues/879 about improving the  experience when trying to view a removed practice
